Pipfile

```diff
@@ -42,6 +42,7 @@ MarkupSafe = "*"
 WTForms = "*"
 jwcrypto = "*"
 email-validator = "*"
+pytest-mock = "*"
 
```

Comment:
I had to update the full pipfile lock, when adding pytest-mock I was getting a strange dependency failure which I couldn't resolve, so had to update everything